Codigo/FileIO.bas

```diff
@@ -1846,16 +1843,15 @@ Sub LoadMapData()
 126         DoEvents
         
 128     Next map
-    
-        'Call generateMatrix(MATRIX_INITIAL_MAP)
-
 130     frmCargando.ToMapLbl.Visible = False
-    
+        Call InitializeInstanceHeap(InstanceMapCount, NumMaps + 1)
```

Comment:
`InstanceManager.InitializeInstanceHeap(InstanceMapCount, NumMaps + 1)` is more clear